docs/customizations/helm.rst

```diff
@@ -421,606 +433,15 @@ NIC Configuration Operator Helm chart customization options can be found `here <
-     - "true"
-     - Restore host drivers when a container
-
-In addition, it is possible to specify any environment variables to be exposed to the NVIDIA DOCA Driver container, such as the standard "HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY".
-
-.. warning::
-   CREATE_IFNAMES_UDEV is set automatically by the Network Operator, depending on the Operating System of the worker nodes in the cluster (the cluster is assumed to be homogenous).
-
-.. warning::
-  When ENABLE_NFSRDMA is set to `true`, it is not possible to load NVME related storage modules from NVIDIA DOCA Driver container when they are in use by the system
-  (e.g the system has NVMe SSD drives in use). User should ensure the modules are not in use and blacklist them prior to the use of NVIDIA DOCA Driver container.
-
-To set these variables, change them into Helm values. For example:
-
-.. code-block:: yaml
-
-    ofedDriver:
-      env:
-      - name: RESTORE_DRIVER_ON_POD_TERMINATION
-        value: "true"
-      - name: UNLOAD_STORAGE_MODULES
-        value: "true"
-      - name: CREATE_IFNAMES_UDEV
-        value: "true"
-
-The variables can also be configured directly via the NicClusterPolicy CRD.
-
-=========================
-RDMA Shared Device Plugin
-=========================
-
-.. list-table::
-   :header-rows: 1
-
-   * - Name
-     - Type
-     - Default
-     - Description
-   * - rdmaSharedDevicePlugin.deploy
-     - bool
-     - `true`
-     - Deploy RDMA shared device plugin.
-   * - rdmaSharedDevicePlugin.image
-     - string
-     - `"k8s-rdma-shared-dev-plugin"`
-     - RDMA shared device plugin image name.
-   * - rdmaSharedDevicePlugin.repository
-     - string
-     - `"ghcr.io/mellanox"`
-     - RDMA shared device plugin image repository.
-   * - rdmaSharedDevicePlugin.resources
-     - yaml
-     - .. code-block:: yaml
-
-          - name: rdma_shared_device_a
-            vendors: [15b3]
-            rdmaHcaMax: 63
-         
-     - The following defines the RDMA resources in the cluster. It must be provided by the user when deploying the chart. Each entry in the resources element will create a resource with the provided <name> and list of devices.
-   * - rdmaSharedDevicePlugin.useCdi
-     - bool
-     - `false`
-     - Enable Container Device Interface (CDI) mode. **NOTE**: NVIDIA Network Operator does not configure container runtime to enable CDI.
-   * - rdmaSharedDevicePlugin.version
-     - string
-     - `"sha-4f3eb2224b8b5f97be3f17441ddee8d41753b7d5"`
-     - RDMA shared device plugin version.
-
-==========================================
-RDMA Device Plugin Resource Configurations
-==========================================
-
-These configurations consist of a list of RDMA resources, each with a name and a selector of RDMA capable network devices to be associated with the resource. Refer to `RDMA Shared Device Plugin Selectors <https://github.com/Mellanox/k8s-rdma-shared-dev-plugin#devices-selectors>`_ for supported selectors.
```

Comment:
Make sure we have similar link/description in the RDMA examples

Reply:
i think this can go under the same section as the comment above RE CDI.

Reply:
Should we have this in the godoc instead?

Reply:
+1

Reply:
Added in godoc